### PR TITLE
Remove spawn of zombie with a flamethrower.

### LINKED ISSUE #62252: Zombie Burners are kind of dumb
### Is your feature request related to a problem? Please describe.

Zombie Burners are dead flamethrower soldiers. Their tank is described as leaking, and it in fact has a limited capacity of napalm it can leak before it stops leaking entirely. In addition to this, they have a rare chance to explode in a lethal conflagration when killed. 

This obviously doesn't make a whole lot of sense. Why do they explode? Does "Blob Magic" ignite the napalm when they die? Napalm tanks exploding is a hollywood/videogame fabrication first of all, it doesn't really conform to real life. Secondly, there is a good chance the napalm tank is entirely empty by the time they would die, but they still explode anyways. Third, the napalm tanks only start leaking once the zombies are in your reality bubble, when in reality their tanks should have emptied within 10 minutes of them resurrecting as a zombie.

Them exploding is very much a "gotcha" mechanic, the player has no way to reasonably expect that they will explode into flames when killed because of how improbable and unrealistic it is. And one explosion can very easily kill a player in melee especially if they are new enough to the game that they don't know that these enemies can erupt into flames on death.

### Solution you would like.

1. Zombie burners don't leak napalm when moving.
2. They don't explode when killed.

### Describe alternatives you have considered.

Remove zombie burners.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Purpose
Flamethrowers aren't fielded by the modern military or police or pretty much anyone. It's not reasonable to run across a zombie wearing one, much less with all the baggage this implementation brings with it (infinite fuel supply that disappears when they die?)

Partially adresses #50248 
Fixes #62252 

#### Implementation
Removes remaining spawns of flamers, moves them to obsolete holding file to avoid breaking saves and mods.

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
The Military Corpse Disposal(which I made to be the profession version of the burner) needs to go too

**Comment by oosyrag** [comment]
Possibly fielded as a specific counter to the opposition?

**Comment by Night-Pryanik** [comment]
Flamethrower units were sort of a desperate action by the government when they realized that zombies ain't killed by firearms, it's noted in their description. These units are not a part of organized military or police units.

**Comment by Night-Pryanik** [comment]
If you're still inclined to get rid of it, please move it to No Hope mod instead. I don't want my hard work be thrown out like this.

**Comment by PapaOscarWhiskey** [comment]
> If you're still inclined to get rid of it, please move it to No Hope mod instead. I don't want my hard work be thrown out like this.

Kevin is an utterly selfish and entitled retard who doesn't give a shit about what others want or like, do you seriously expect him to listen to your pleads?

**Comment by oosyrag** [comment]
I generally don't expect anything actually, but not stating a reasonable opposing opinion that may have been overlooked just because I expect it to not to result anything would be even more idiotic. It doesn't cost anything to have a differing opinion rejected unless the one making it has overinflated sensitive ego. All of our time here is volunteered anyway, and this isn't our house.

**Comment by kevingranade** [comment]
> please move it to No Hope mod instead.

That's what I did? It's in he obsoletion file so mods can stil use it.

**Comment by Night-Pryanik** [comment]
You also removed it from monster group. To make it 
accessible in the mod, it will require re-adding it in mod's monster group, which I'm asking you for. Sure, I could do it myself later, I'm asking for it just because you're working with it now.

File: data/json/obsoletion/professions.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+]
```
Comment (TheShadowFerret) [inline data/json/obsoletion/professions.json:1]:
```suggestion
[
```

**Comment by kevingranade** [comment]
> 

Aha, sure no problem.

**Pull request merged**